docs/src/content/docs/de/components/asides.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Nebenbemerkungen
+description: Lerne, wie du in Starlight zusätzliche Informationen neben dem Hauptinhalt einer Seite anzeigen kannst.
+---
+
+import { Aside } from '@astrojs/starlight/components';
+
+Um sekundäre Informationen neben dem Hauptinhalt einer Seite anzuzeigen, verwende die Komponente `<Aside>`.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Aside slot="preview">
+    Füge nicht wesentliche, ergänzende Informationen in ein `<Aside>`.
+</Aside>
+
+</Preview>
+
+## Import
+
+```tsx
+import { Aside } from '@astrojs/starlight/components';
+```
+
+## Verwendung
+
+Zeige eine Nebenbemerkung (auch bekannt als „Hinweise“, „Ermahnungen“ oder „Aufrufe“) mit Hilfe der Komponente `<Aside>`.
+
+Ein `<Aside>` kann ein optionales [`Typ`](#type) Attribute haben, welches die Farbe, das Symbol und den Standardtitel der Nebenbemerkung steuert.
```

Comment:
I think here we should probably avoid translating "type" as it's the name of the prop that people will need to use:

```suggestion
Ein `<Aside>` kann ein optionales [`type`](#type) Attribute haben, welches die Farbe, das Symbol und den Standardtitel der Nebenbemerkung steuert.
```